From b12c3f5de6340b997e5a2992710828f3e89308a0 Mon Sep 17 00:00:00 2001
From: Simon Reinhardt <simon.reinhardt@stud.uni-regensburg.de>
Date: Mon, 26 Jun 2017 14:57:32 +0200
Subject: [PATCH] Fix #65: No quoting for output filenames.

Insert missing call to quote_escape (escape backslashes and such for gnuplot
double-quote strings).
---
 lib/PDL/Graphics/Gnuplot.pm | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/PDL/Graphics/Gnuplot.pm b/lib/PDL/Graphics/Gnuplot.pm
index 0edf7ef..ff96cc9 100644
--- a/lib/PDL/Graphics/Gnuplot.pm
+++ b/lib/PDL/Graphics/Gnuplot.pm
@@ -5188,6 +5188,7 @@ our $pOptionsTable =
 			  if($vv ne $v) {
 			      carp "INFO: Plotting to '$vv'\n";
 			  }
+			  $vv = quote_escape($vv);
 			  return "set $k \"$vv\"\n";
 		    },
 		    undef,3,
